tests/wip/simple-e2e.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
I'd rather not have a WiP directory here. Are you able to split this off into another branch?
(Sorry if it sounds like I'm being a diva) :)
This is all really good.

Reply:
Just deleted the WIP files and finished the logic implementation within the folder. Not it is added to the PR